frontend/src/tests/selectors/mineSelectors.spec.js

```diff
@@ -1,29 +1,118 @@
+  it('`getMineGuid` calls `mineReducer.getMineGuid` when `storeMineList` is dispatched', () => {
+    const storeAction = storeMineList(mockResponse);
+    const storeState = mineReducer({}, storeAction);
+    const mockState = {
+      [MINES]: storeState
+    };
+    expect(getMineGuid(mockState)).toEqual(mineGuid);
+  });
+
+  it('`getMineGuid` calls `mineReducer.getMineGuid` when `storeMine` is dispatched', () => {
+    const storeAction = storeMine(mineResponse, "18145c75-49ad-0101-85f3-a43e45ae989a");
+    const storeState = mineReducer({}, storeAction);
+    const mockState = {
+      [MINES]: storeState
+    };
+    expect(getMineGuid(mockState)).toEqual("18145c75-49ad-0101-85f3-a43e45ae989a");
   });
 
   it('`getMineIds` calls `mineReducer.getMineIds`', () => {
-    const storeAction = storeMineList(mineListInput);
+    const storeAction = storeMineList(mockResponse);
+    const storeState = mineReducer({}, storeAction);
+    const mockState = {
+      [MINES]: storeState
+    };
+    expect(getMineIds(mockState)).toEqual(mineIds);
+  });
+
+  it('`getMineNames` calls `mineReducer.getMineNames`', () => {
+    const storeAction = storeMineNameList(Mock.MINE_NAME_LIST);
     const storeState = mineReducer({}, storeAction);
     const mockState = {
       [MINES]: storeState
     };
+    expect(getMineNames(mockState)).toEqual(mineNameList);
+  });
+
+  it('`getMineStatusOptions` calls `mineReducer.getMineStatusOptions`', () => {
+    const storeAction = storeStatusOptions(Mock.STATUS_OPTIONS);
+    const storeState = mineReducer({}, storeAction);
+    const mockState = {
+      [MINES]: storeState
+    };
+    expect(getMineStatusOptions(mockState)).toEqual(mineStatusOptions);
+  });
+
+  it('`getMineRegionOptions` calls `mineReducer.getMineRegionOptions`', () => {
+    const storeAction = storeRegionOptions(Mock.REGION_OPTIONS);
+    const storeState = mineReducer({}, storeAction);
+    const mockState = {
+      [MINES]: storeState
+    };
+    expect(getMineRegionOptions(mockState)).toEqual(mineRegionOptions);
+  });
+
+  it('`getMinesPageData` calls `mineReducer.getMinesPageData`', () => {
+    const storeAction = storeMineList(Mock.PAGE_DATA);
+    const storeState = mineReducer({}, storeAction);
+    const mockState = {
+      [MINES]: storeState
+    };
+    expect(getMinesPageData(mockState)).toEqual(minesPageData);
+  });
+
+  it('`getCurrentPermittees` calls `mineReducer.getCurrentPermittees`', () => {
+    mines = Mock.MINES.mines;
+    mineGuid = Mock.MINES.mineIds[1]
+    const selected = getCurrentPermittees.resultFunc(mines, mineGuid);
+    expect(selected).toEqual((Mock.PERMITTEE.permittees));
+  });
 
-    expect(getMineIds(mockState)).toEqual(["test123", "test456"]);
+  it('`getCurrentPermitteeIds` calls `mineReducer.getCurrentPermitteeIds`', () => {
+    mines =  Mock.MINES.mines;
+    mineGuid = Mock.MINES.mineIds[1]
+    const selected = getCurrentPermitteeIds.resultFunc(mines, mineGuid)
```

Comment:
Semicolons.

Okay. I thiiiink I see how these tests are working.. They essentially just test that `getCurrentPermitteeIds` returns the expected value, which infers that `.getCurrentPermitteeIds` was called in the process?